02-bookkeeping.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: First Steps in LHCb
+subtitle: Finding data to process
+minutes: 10
+---
+> ## Learning Objectives {.objectives}
+>
+> * Find MC in the book-keeping
+> * Find data in the book-keeping
+> * Find the decay you want
+> * check reco
+> * ignore what (flagged) stripping is used
+> * run over those files with the strip21 module you want
+> * nTuples!
+
+To feed simulated and/or real data to your DaVinci job you first
+need to locate some events. In this tutorial we will use the
+decay D* -> D(->K pi)pi as an example.
+
+Navigate to the [book-keeping](https://lhcb-portal-dirac.cern.ch/DIRAC/?view=tabs&theme=Grey&url_state=1|*LHCbDIRAC.BookkeepingBrowser.classes.BookkeepingBrowser:,) which lets you find both simulated
+and real data.
+
+At the bottom of the "Bookkeeping tree" tab there is a dropdown menu
+labelled `Simulation Condition`, open it and change it to `Event
+type`.
+
+We will analyse 2012 data, and correspondigly use simulation for 2012
+data. To navigate to the simulation expand the folder icon in the
+"Bookkeeping tree" window. Navigate to the `MC/2012` folder. This will
+give you a very long list of all possible decay types for which there
+is simulated data. We are looking for a folder which is named
+`27163003 (Dst_D0pi,Kpi=DecProdCut)`. The number is a numerical
+representation of the [event
+type](https://cds.cern.ch/record/855452?ln=en).  The text is the human
+readable version of that.
+
+This sample of simulated events will only contain decays of the type
+D* -> D(-> K pi)pi.
```

Comment:
How pedantic do we want to be here? Technically, each generated event will contain a signal decay (as a set of MC particles) that passes the generator level cuts _and_ the rest of the event. Each event might not contain the signal in the reconstruction.